src/EnergyPlus/ZoneEquipmentManager.cc

```diff
@@ -2656,7 +2716,7 @@ namespace ZoneEquipmentManager {
                                 }
                                 ShowContinueError("...check zone thermostat set point and design supply air temperatures");
                                 ShowContinueError("...zone name = " + CalcFinalZoneSizing(I).ZoneName);
-                                ShowContinueError("...design heating load         = " + RoundSigDigits(CalcFinalZoneSizing(I).DesCoolLoad, 2) + " W");
+                                ShowContinueError("...design heating load         = " + RoundSigDigits(CalcFinalZoneSizing(I).DesHeatLoad, 2) + " W");
                                 ShowContinueError("...thermostat set point temp   = " + RoundSigDigits(CalcFinalZoneSizing(I).HeatTstatTemp, 3) +
```

Comment:
The merge conflict was just that you had changed DesCoolLoad to DesHeatLoad, and a different branch had fixed a typo from piont to point.  Easy enough.  I'll let CI have another pass over this updated version.